### ISSUE TITLE
BeOp request to not be globally blocked

### ISSUE THREAD

**Issue by sebrobert** [SystemJargon/blocklists#1]
Hello, I am opening that issue because we, at **BeOp**, are a **global content management system** that allows both _Publishers_ and _Advertisers_ to create contents / creatives. 

We have two types of contents → 

- editorial 
- sponsored 

Basically, if our domain is globally blocked by your list, even the editorial contents will be blocked (these ones are made by publisher to animate their pages in a full classic way, helping people or providing people a service or game or whatever on their pages). The web user will experiment a poor navigation experience as they will not even find the publisher contents in pages. Consider that somehow, we are a kind of **CMS provider** to create interactive contents. 

On the other way, we also are provider of interactive sponsored contents, and **we understand that you might want to block us** in your ad blocking list. So, to not impact our publishers, and to let you block all our advertising contents, we have created separated endpoints **to isolate the delivery of editorial and sponsored contents** 

Please, can you **NOT** block 
- contents.beop.io ✅ 
- contents-tracking.beop.io ✅ 
- t.beop.io ✅ 

On the other way, you can block 
- ads.beop.io ⛔ 
- ads-tracking.beop.io ⛔ 
- s.beop.io ⛔ 

Please feel free to contact me if you need any other information, or if you have any request for us to improve the web experience for end users ! 
Best, 
Sébastien Robert

**Comment by SystemJargon** [comment]
We are unable to locate any such blocking of provided hostnames within this repo to a particular list (as of the time of writing). 

We have noted the hosts you provided, these are **now** added to one of the lists in this repo for blocking because content analysis matches either, "Tracking", "Telemetry" or "Advertising".

This can be reviewed again if you wish to open a new issue, however we will do further content analysis scans.
